apps/playground/src/app/modules/components/header-module/header-module.component.html

```diff
@@ -89,6 +89,12 @@ <h2>DsfrHeaderComponent</h2>
       type="It takes a boolean">
       It will display the autocompletion of the search
     </play-input-property-definition>
+    <play-input-property-definition
+      name="miniMobileHeader"
+      default="false"
+      type="It takes a boolean">
+      It will display a minimalistic header on mobile if set to true.
```

Comment:
It will display a minimalistic header on mobile if set to true. Note that
      the display will stay the same when on desktop. Among the changes, be
      aware that both the operator logo and the app description will NOT be
      displayed when on mobile, even if they are provided.